applications/nrf_desktop/doc/bas.rst

```diff
@@ -8,11 +8,9 @@ Use the ``bas`` module to enable and define the GATT Battery Service, and notify
 Module Events
 *************
 
-+---------------------+-------------------------+-------------+---------------+------------------+
-| Source Module       | Input Event             | This Module | Output Event  | Sink Module      |
-+=====================+=========================+=============+===============+==================+
-| ``battery_meas``    | ``battery_level_event`` | ``bas``     |               |                  |
-+---------------------+-------------------------+-------------+---------------+------------------+
+.. include:: generated/event_propagation/bas.rst
```

Comment:
Question: how can we, tehc writers, or anyone else update these tables in the future, if they need updates?